inc/helpers.php

```diff
@@ -23,7 +23,18 @@ function primer_get_the_page_title() {
 
 		case is_front_page() :
 
-			$title = get_the_title( get_option( 'page_on_front' ) );
+			/**
+			 * Filter the page title when the front page is set to display latest posts.
+			 *
+			 * Settings > Reading > Front page displays > Your latest posts
+			 *
+			 * @since NEXT
+			 *
+			 * @var string
+			 */
+			$latest_posts_title = (string) apply_filters( 'primer_latest_posts_title', '' );
```

Comment:
@EvanHerman What do you think about this?

<img width="146" alt="screen shot 2017-02-15 at 12 10 20 pm" src="https://cloud.githubusercontent.com/assets/522158/22988486/e870d262-f377-11e6-805a-21ad54debfe5.png">